scopes/harmony/aspect-loader/plugins.ts

```diff
@@ -53,9 +56,20 @@ export class Plugins {
     return aspect;
   }
 
+  async loadModule(path: string) {
+    NativeCompileCache.uninstall();
```

Comment:
We should just stop using the native compile cache, it doesn't make sense once we start going full esm